packages/google_sign_in/google_sign_in_ios/darwin/google_sign_in_ios.podspec

```diff
@@ -16,10 +16,11 @@ Enables Google Sign-In in Flutter apps.
   s.public_header_files = 'Classes/**/*.h'
   s.module_map = 'Classes/FLTGoogleSignInPlugin.modulemap'
 
-  # AppAuth is a transitive dependency of GoogleSignIn.
-  # Depend on 1.7.4 or higher, which defines modules.
-  s.dependency 'AppAuth', '~> 1.7.4'
-  s.dependency 'GoogleSignIn', '~> 7.1'
+  # AppAuth and GTMSessionFetcher are GoogleSignIn transitive dependencies.
+  # Depend on versions which defines modules.
+  s.dependency 'AppAuth', '>= 1.7.4'
+  s.dependency 'GTMSessionFetcher', '>= 3.4.0'
+  s.dependency 'GoogleSignIn', '>= 7.1'
```

Comment:
Isn't `~> 7.1` right for this one? We don't want to get 8.x without opting in because we do call its APIs directly.

Reply:
My brain refuses to retain what `~>` does.  Done.

Reply:
For the future: we could consider just never using it, since we all have to look it up every time, and manually do the ranges ourselves. But hopefully in the medium term we won't need podspecs at all.